.github/workflows/cache.yml

```diff
@@ -0,0 +1,54 @@
+name: Cache stuff
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  deployment:
+    runs-on: macos-latest
```

Comment:
Is there a reason for macos-lastest instead of ubuntu-latest? Just to know

Reply:
Unfortunately ubuntu-latest in regular GitHub Actions runners, does not support nested virtualization. This feature is only available for free on macOS.
https://github.com/actions/runner-images/discussions/7191#discussioncomment-6145657
Since macOS still uses bash I thought it could be a starting point, also considering that the focus of this CI workflow is to test if the Ansible rules are executed properly on the given guest machine, so the outcome is pretty much the same as if we were able to use ubuntu-latest.